CHANGELOG.md

```diff
@@ -1,126 +1,22 @@
-  - Added "Vanilla Weapon Cycle" option.
-  - Added "Last" weapon key binding.
-  - Added support for SSG key binding when using vanilla compatibility level.
-  - Improved performance when handling many input events at once.
-  - Optimized input polling for older computers (thanks to @gendlin).
-  - Added support for native keyboard input (from Chocolate Doom).
-
-* [Gamepad](https://github.com/fabiangreffrath/woof/wiki/Gamepad):
-  - Added custom weapon slots: the default configuration uses the D-pad and matches the Doom II Xbox 360 port.
-  - Added rumble support. Rumble patterns are generated dynamically from sound files. Supports weapon mods.
-  - Added gyro support.
-  - Added flick stick support.
-  - Added option to force platform-specific (i.e. Xbox, Playstation, or Switch) button names: see `joy_platform` in `woof.cfg`.
-  - Added option to force keyboard or gamepad menu help text, useful for mixed input devices: see `menu_help` in `woof.cfg`.
-  - Streamlined save game menu when using a gamepad.
-
-* Quality of Life:
-  - Added "Auto Save" option: the game is automatically saved at the beginning of a map, after completing the previous one.
-  - Added compatibility database from DSDA-Doom (`COMPDB` lump), which automatically applies compatibility options.
-  - Added option to correct the automap's aspect ratio (thanks to @thearst3rd).
-  - Added aliases for popular command-line parameters (`-cl`, `-nomo`, `-uv`, `-nm`).
-  - Added announcement of map titles (from DSDA-Doom).
-  - Added Crispy Doom automap color scheme.
-  - Added "count" option for secret-revealed message (thanks to @MrAlaux).
-  - Added level stats widget format settings (thanks to @MrAlaux).
-  - Added fast exit when pressing Alt+F4 or the window close button.
-  - Improved deathmatch HUD widgets (scoreboard and level time countdown).
-
-* Tools:
-  - Added advanced coordinates widget (from DSDA-Doom).
-  - Added command history widget (from DSDA-Doom).
-
-* Miscellaneous:
-  - Replaced "Show ENDOOM Screen" menu item with "Exit Sequence" (Off, Sound Only, PWAD ENDOOM, Full).
-  - Added drag-and-drop on non-Windows systems.
-  - Added `SPEED` cheat to show a speedometer.
-  - Rewrote help strings in `woof.cfg` (thanks to @MrAlaux).
-  - Added interpolation of wipe melt animation (from Rum and Raisin Doom).
-  - Fit finale text to widescreen instead of inserting line breaks.
-  - Always use traditional menu ordering (removed Boom choice).
-  - Moved all autoload and internal resources to `woof.pk3` archive.
-  - Allowed setting `gamedescription` (and thus the window title) with DEHACKED.
-  - Allowed up to 10 episodes in UMAPINFO.
-  - Added more paths to find IWADs.
-  - Removed source port credits screen, restoring the vanilla demo loop.
+* Process TRAKINFO in IWADs. Compatibility with Chex Quest 3: Vanilla Edition 1.0.
```

Comment:
Chex 3 Vanilla 1.0 doesn't actually have a TRAKINFO in the IWAD, the digital TRAKINFO music is actually a separate WAD to be loaded on top of the IWAD.

Reply:
Fixed in 107ad46

@elf-alchemist send me a version with TRAKINFO, it's Bethesda.net only?

Reply:
Hi, yes it was part of the testing I was doing for the BNet version that I happened to come across that it wasn't loading the `TRAKINFO` in an IWAD (I thought it'd be a shame to entirely exclude all possible custom IWADs in the future from having it built-in) but also the fact that I tend to merge and customize the PWADs in my local setup for a more concise package :P